projects/step-frontend/styles/components/_ts-tooltip.scss

```diff
@@ -57,4 +93,48 @@
       font-size: 18px;
     }
   }
+
+  .link-icon {
+    margin-left: 0.6rem;
+    box-sizing: border-box;
+    position: relative;
+    display: block;
+    width: 16px;
```

Comment:
Use `rem` instead of `px`

Reply:
@dvladir this is a hardcoded icon. rem would not work ok.

Reply:
@lucianbaciu ok noted